web/impact/impact/v0/views/startup_detail_view.py

```diff
@@ -84,7 +82,9 @@ def _calc_result(self):
             "twitter_handle": startup.twitter_handle,
             "website_url": startup.website_url,
 
-            "image_token": encrypt_image_token(startup.high_resolution_logo),
+            "image_token": encrypt_image_token(
+                startup.high_resolution_logo.name) if
+            startup.high_resolution_logo else '',
```

Comment:
is the if statement needed now since None passed to encrypt_image_token returns empty string